Tasks/Gradle/CodeAnalysis/Common/BuildOutput.ts

```diff
@@ -59,14 +54,12 @@ export class BuildOutput {
     private getBuildDirectoryName(): string {
         switch (this.buildEngine) {
             case BuildEngine.Gradle:
-                {
-                    return 'build';
-                }
+                return 'build';
```

Comment:
I suggest we set the version 1.112.0 instead of bumping to 2.0 because the refactorings are many, but not significant in functionality (that I could see) and it has been thoroughly tested and retested.